compiler/semcall.nim

```diff
@@ -327,10 +328,30 @@ proc resolveOverloads(c: PContext, n, orig: PNode,
       pickBest(callOp)
 
     if overloadsState == csEmpty and result.state == csEmpty:
+      var msg = ""
```

Comment:
This should be extracted into a helper proc.

Reply:
done